### ISSUE TITLE
Total balance on wallets page

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#2618]
### Discussed in https://github.com/btcpayserver/btcpayserver/discussions/2306

<div type='discussions-op-text'>

<sup>Originally posted by **britttttk** February 21, 2021</sup>
Show the total balance of all the wallets on the wallets page. 

This would be nice on the future dashboard feature.. but can we add it here for now?</div>

**Comment by dstrukt** [comment]
Yeah we can, we only want to list the total number of bitcoin between all of the wallets, correct? 

I ask explicitly, as we're not showing an accompanying fiat current as well, like so...

1.24303290 BTC
$42,04.02 

Thoughts?

**Comment by britttttk** [comment]
> total number of bitcoin between all of the wallets 
ack

**Comment by bolatovumar** [comment]
One thing we need to consider here is that you may have many wallets with different coins, not just BTC. In this case we would have to come up with a way to show the total balance across different coins. Not sure how the UI would work for that.

**Comment by pavlenex** [comment]
ping @dstrukt @dennisreimann @Zaxounette perhaps we can add to the agenda for one of the next meetings, displaying total balance for different currencies.
